types/store/plugins.ts

```diff
@@ -85,6 +85,13 @@ export type FileDropdownPluginComponent = {
     action: (fileInfo: FileInfo) => void;
 };
 
+export type UserGuideDropdownPluginComponent = {
```

Comment:
Can we add this to the `PluginsState` type defined at the top of this file? Ideally all of the plugin components are typed there, but we currently have a catch-all `[componentName: string]: PluginComponent[];` that can sometimes defeat the purpose of using types.